docs/game_sdk/Applications.md

```diff
@@ -21,6 +21,15 @@ This manager also includes a couple useful helper functions, like getting the lo
 | Scopes      | string | a list of oauth2 scopes as a single string, delineated by spaces like `"identify rpc gdm.join"` |
 | Expires     | Int64  | the timestamp at which the token expires                                                        |
 
+###### SignedAppTicket Struct
+
+| name           | type                                                                                                                     | description                                              |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------- |
+| application_id | Int64                                                                                                                    | the application id for the ticket                        |
+| user           | [User](#DOCS_GAME_SDK_USER/data-models-user-struct)                                                                      | the user for the ticket                                  |
+| entitlements   | list of partial [Entitlements](#DOCS_GAME_SDK_STORE/data-models-entitlement-struct) structs that contain just the SKU id | the list of the user's entitlements for this application |
+| timestamp      | string                                                                                                                   | the ISO timestamp for the ticket                         |
```

Comment:
This timestamp is in ISO8601 format, correct? Might be a good idea to explicitly label it as an ISO8601 timestamp, not an "ISO timestamp".